benchmarks/run.py

```diff
@@ -151,6 +147,50 @@ class RunResult:
 }
 
 
+KERNEL_METRIC_MAPPINGS: dict[str, dict[str, str]] = {
+    "vector_add": {
+        "triton_add-speedup": "triton_speedup",
+        "triton_add-accuracy": "triton_accuracy",
+        "torch_compile_add-speedup": "torch_compile_speedup",
+        "torch_compile_add-accuracy": "torch_compile_accuracy",
+        "helion_add-speedup": "helion_speedup",
+        "helion_add-accuracy": "helion_accuracy",
+    },
+    "vector_exp": {
+        "triton_exp-speedup": "triton_speedup",
+        "triton_exp-accuracy": "triton_accuracy",
+        "torch_compile_exp-speedup": "torch_compile_speedup",
+        "torch_compile_exp-accuracy": "torch_compile_accuracy",
+        "helion_exp_tritonbench-speedup": "helion_speedup",
+        "helion_exp_tritonbench-accuracy": "helion_accuracy",
+    },
+    "sum": {
+        "triton_sum-speedup": "triton_speedup",
+        "triton_sum-accuracy": "triton_accuracy",
+        "torch_compile_sum-speedup": "torch_compile_speedup",
+        "torch_compile_sum-accuracy": "torch_compile_accuracy",
+        "helion_sum_tritonbench-speedup": "helion_speedup",
+        "helion_sum_tritonbench-accuracy": "helion_accuracy",
+    },
+    "layer_norm": {
+        "triton_layer_norm-speedup": "triton_speedup",
+        "triton_layer_norm-accuracy": "triton_accuracy",
```

Comment:
I wonder for layer_norm if we should use `liger_layer_norm` instead of the triton version (liger version is about 20% faster)